webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/CellEditor.tsx

```diff
@@ -108,7 +110,7 @@ export const CellEditor = observer<Pick<RenderEditCellProps<IResultSetRowKey>, '
         onMouseDown={preventClick}
         onMouseUp={preventClick}
       >
-        {
+        {editorPortal &&
           createPortal(
```

Comment:
is there a chance that portal creates on each render of this component?

Reply:
I suggest to move portal to `ref` and once `editorPortal` is not `null`, reassign this ref to `createPortal` value